src/shared/ui/components/InstructionsTable.vue

```diff
@@ -45,36 +44,50 @@ function getInstructionPayloadEntity(item: Instruction) {
   return Object.entries(item.payload)[0][0];
 }
 
-const instructionsTable = useTable(http.fetchInstructions);
-
-async function fetchInstructions() {
-  try {
-    if (isOnAccountPage.value) {
-      await instructionsTable.fetch({
-        ...objectOmit(listState, shouldShowKind.value ? ['kind'] : []),
-        transaction_status: listState.status,
-      });
-    } else
-      await instructionsTable.fetch({
-        ...objectOmit(listState, shouldShowKind.value ? ['kind'] : []),
-      });
-  } catch (e) {
-    handleUnknownError(e);
-  }
-}
-
 const isOnAccountPage = computed(() => props.filterBy.kind === 'authority');
+const shouldShowKind = computed(() => listState.kind === 'All');
 
 const listState = reactive({
-  status: null,
-  authority: computed(() => props.filterBy.kind === 'authority' && props.filterBy.value.toString()),
-  transaction_hash: computed(() => props.filterBy.kind === 'transaction' && props.filterBy.value),
+  transaction_status: null,
+  authority: computed(() => (props.filterBy.kind === 'authority' ? props.filterBy.value.toString() : '')),
+  transaction_hash: computed(() => (props.filterBy.kind === 'transaction' ? props.filterBy.value : '')),
   kind: 'All' as ftm.TabInstructions,
+  page: 1,
+  per_page: 10,
 });
 
-const shouldShowKind = computed(() => listState.kind === 'All');
+async function fetchInstructions(params: Reactive<typeof listState>) {
```

Comment:
I'd suggest removing this function.

Instead, compute the actual params reactively:

```ts
const params = computed<InstructionSearchParams>(() => ...)
```

and setup the scope with them directly:

```ts
useParamScope(
  () => ({ key: JSON.stringify(params.value), payload: params }),
  ({ payload }) => handleParamScope(() => http.fetchInstructions(payload))
)
```

This way it could turn out that the whole `listState` is extra.